kafka-bom/build.gradle.kts

```diff
@@ -4,7 +4,7 @@ plugins {
 }
 
 
-var kafkaVersion = "7.4.0"
+var kafkaVersion = "7.5.1"
```

Comment:
Why not latest `7.5.2`?
https://packages.confluent.io/maven/org/apache/kafka/kafka-streams/7.5.2-ccs/

Reply:
Didn't find that in mvnrepository tab - https://mvnrepository.com/artifact/org.apache.kafka/kafka-streams?repo=confluent-packages